ci/make_build_bin.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+. ci/build_consts.sh
```

Comment:
This is relative to the initial bash invocation, right? Should this form a path relative to the path of the current script?

Reply:
I think right now all scripts assume they're being run from $REPO_TOP. I'd like to get to a point where this is manually enforced... but this is one of many problematic file references.